js/counties.js

```diff
@@ -0,0 +1,73 @@
+function hideCounties() {
+  map.selectAll('.county')
+    .data([])
```

Comment:
well, it seems to work, and made sense to me because if the data array is empty then you'd figure everything would be in the .exit() selection, right? but i suppose an alternative would be to simply `map.selectAll('.county').remove()` - yep, just tried it and it works fine. i'll switch to that.